### ISSUE TITLE
Submitting any changes leads to broken redirect

### ISSUE THREAD

**Issue by Zer0ot** [evekb/evedev-kb#37]
If i Change anything in the configuration and hit the Save button, changes are saved properly, but the redirect I'm getting is broken. For example:

```
https://my.domain.at/killboard/index.php?a=admin&field=Advanced&sub=Configuration
```

leads me after saving to:

```
https://my.domain.atkillboard/index.php?a=admin&field=Advanced&sub=Configuration
```

The / between Domain and Directory is gone. The Result is, I'm getting redirected to my browsers "Server not found" Page. The Main Webpage Link is set properly with trailing slash, the Killboard Host is set without trailing slash, of course (would lead to URls with 2 slashes, but won't fix this issue.) it's every time the slash between Domain and First Directory that is missing. everytime only after submitting any Forms in Administration (didn't tried out comments by now)

**Comment by Salvoxia** [comment]
Hi,

while looking at the admin code I found that the there's a reload functionality, that hardcodedly directs to http://...
Is it possible your :80 -> :443 redirection has anything to do with it?
You could test this by changing the hardcoded http:// to https://. 
I'll definitely see to it that the reload will handle the https scheme correctly, but I'd like to know if that also solves your issue.

For testing this out, open common/admin/option_config.php, find the following piece of code (around line 488):

``` php
header("Location: http://".$_SERVER['HTTP_HOST']
                .$_SERVER['SCRIPT_NAME'].'?'.$_SERVER['QUERY_STRING']);
```

Simple change that to https:

``` php
header("Location: https://".$_SERVER['HTTP_HOST']
                .$_SERVER['SCRIPT_NAME'].'?'.$_SERVER['QUERY_STRING']);
```

Regards,
Salvoxia

**Comment by Zer0ot** [comment]
Totally solves the Issue for me! Top!
Best Regards, Haymo
